daemons/pacemakerd/pacemakerd.c

```diff
@@ -1236,10 +1237,14 @@ main(int argc, char **argv)
     mainloop_add_signal(SIGTERM, pcmk_shutdown);
     mainloop_add_signal(SIGINT, pcmk_shutdown);
 
-    if (running_with_sbd) {
+    if ((running_with_sbd) && pcmk__get_sbd_sync_resource_startup()) {
         pacemakerd_state = XML_PING_ATTR_PACEMAKERDSTATE_WAITPING;
         startup_trigger = mainloop_add_trigger(G_PRIORITY_HIGH, init_children_processes, NULL);
     } else {
+        if (running_with_sbd) {
+            crm_warn("Enabling SBD_SYNC_RESOURCE_STARTUP would improve "
+                     "reliability of interworking between SBD & pacemaker.");
```

Comment:
I'm thinking we should add "(if supported by your sbd version) would improve" so people don't blindly add it

Reply:
Tried to keep the line short. But as it is already > 80 chars more clarity makes probably sense.